src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts

```diff
@@ -331,9 +343,21 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
       .pipe(takeUntil(this.destroy$))
       .subscribe((response) => {
         response.ok ? this.matSnackBar.snackType.changesSaved() : this.matSnackBar.snackType.error();
-        this.getOrderInfo(this.orderId);
+        if (response.ok) {
+          this.getOrderInfo(this.orderId);
+          for (const key in changedValues.generalOrderInfo) {
```

Comment:
Don't use for...in
const { generalOrderInfo } = changedValues;
Object.keys(generalOrderInfo).forEach((key: string) => {
    const value = generalOrderInfo[key];
    if (value) {
        this.postDataItem([this.orderId], key, value);
    }
})

Reply:
done